hammers/scripts/reservation_usage_notification.py

```diff
@@ -0,0 +1,192 @@
+  font-weight: 300;
+  src: local('Open Sans Light'), local('OpenSans-Light'), url(http://fonts.gstatic.com/s/opensans/v13/DXI1ORHCpsQm3Vp6mXoaTa-j2U0lmluP9RWlSytm3ho.woff2) format('woff2');
+  unicode-range: U+0460-052F, U+20B4, U+2DE0-2DFF, U+A640-A69F;
+}}
+.body {{
+    width: 90%;
+    margin: auto;
+    font-family: 'Open Sans', 'Helvetica', sans-serif; 
+    font-size: 11pt;
+    color: #000000;
+}}
+a:link {{ color: #B40057; text-decoration: underline }}
+a:visited {{ color: #542A95; text-decoration: none }}
+a:hover {{ color: #B40057; background-color:#C4FFF9; text-decoration: underline }}
+</style>
+
+<div class="body">
+<p>Dear {{{{ vars['username'] }}}},</p>
+<br>
+
+{email_body}
+
+<br>
+<p><i>
+This is an automatic email, please <b>DO NOT</b> reply! 
+If you have any question or issue, please submit a ticket on our <a href='https://www.chameleoncloud.org/user/help/' target='_blank'>help desk</a>.
+</i></p>
+
+<br><br>
+<p>Thanks,</p>
+<p>Chameleon Team</p>
+
+</div>
+<br><br>
+'''
+
+RESERVATION_START_EMAIL_BODY = '''
+<p>We're sending this email to remind you that your lease {{ vars['leasename'] }} (ID: {{ vars['leaseid'] }}) under project {{ vars['projectname'] }} on {{ vars['site'] }} 
+will start on {{ vars['startdatetime_utc'] }} UTC / {{ vars['startdatetime_ct'] }} Central Time.</p>
+'''
+
+IDLE_RESERVATION_EMAIL_BODY = '''
+<p>We're sending this email to inform you that your lease {{ vars['leasename'] }} (ID: {{ vars['leaseid'] }}) under project {{ vars['projectname'] }} on {{ vars['site'] }} 
+has idled for more than {{ vars['idlehours_threshold'] }} hours.</p>
+
+<p>If you don't need the reservation anymore, please delete the lease using 
+either the Chameleon <a href='https://chameleoncloud.readthedocs.io/en/latest/technical/gui.html#reservations' target='_blank'>web interface</a> 
+or <a href='https://chameleoncloud.readthedocs.io/en/latest/technical/cli.html' target='_blank'>command line interface</a> to release the resources.</p>
+'''
+
+IDLE_HOUR_THRESHOLD = 24
+
+def render_template(email_body, **kwargs):
+    ''' renders a Jinja template into HTML '''
+    templ = Environment(autoescape=select_autoescape(default_for_string=True)).from_string(EMAIL_TEMPLATE.format(email_body=email_body))
+    return templ.render(**kwargs)
+ 
+def send_email(email_host, to, sender, subject=None, body=None):
+    # convert TO into list if string
+    if type(to) is not list:
+        to = to.split()
+    to_list = filter(None, to) # remove null emails
+    
+    msg = MIMEMultipart('alternative')
+    msg['From'] = sender
+    msg['Subject'] = subject
+    msg['To'] = ','.join(to)
+    msg.attach(MIMEText(body, 'html'))
+
+    # send email
+    server = smtplib.SMTP(email_host, timeout=30)
+    server.sendmail(sender, to_list, msg.as_string())
+    server.quit()
+    
+def get_reservations_start_next_day(db):
+    advanced_reservations = query.get_advance_reservations(db)
+    results = []
+    for obj in advanced_reservations:
```

Comment:
There are still two occurrences of `advanced_reservations` instead of `advance_reservations`.